modules/OCPP201/OCPP201.cpp

```diff
@@ -387,6 +387,54 @@ void OCPP201::init_evse_ready_map() {
     }
 }
 
+void OCPP201::init_evses() {
+
+    if (this->r_kvs->call_exists("0")) {
```

Comment:
This should have some kind of prefix, since the kvs modules (at the moment) do not do any sort of automatic prefixing based on their user (which we should probably add there soon)